grbl.h

```diff
@@ -101,6 +101,7 @@
 #define CMD_EXIT 0x03 // ctrl-C (ETX)
 #define CMD_REBOOT 0x14 // ctrl-T (DC4) - only acted upon if preceded by 0x1B (ESC)
 #define CMD_RESET 0x18 // ctrl-X (CAN)
+#define CMD_RESET1 '|' // Reset via Bootloader
```

Comment:
the na,me `CMD_RESET1` doesn't make sense, Please rename to more meaningful name